setup.py

```diff
@@ -75,16 +76,19 @@ def run(self):
 pythonPackages.append('ganga/GangaGUI')
 
 
-install_requires=[
+install_requires = [
     'ipython>=5.0.0',
     'jedi==0.17.2',
```

Comment:
Can we avoid these fixed version?

Reply:
Sure, just made these versions the minimum required dependencies. Seems to be working fine with the newer library versions